configs/experiment/biofoundation/fewshot/ghani.yaml

```diff
@@ -0,0 +1,76 @@
+# @package _global_
+#package global is neccessary!
+#package global is neccessary!
+defaults:
+  - override /logger: wandb_embedding.yaml
+  - override /datamodule: beans_embedding.yaml
+  - override /module: embedding.yaml
+  - override /callbacks: default.yaml
+  - override /trainer: single_gpu.yaml
+  - override /paths: default.yaml
+  - override /hydra: default.yaml
+  - override /datamodule/transforms: embedding_transform.yaml # Here the audio is transformed to a spectrogram
+  - override /datamodule/transforms/waveform_augmentations: none.yaml # No augmentations for now
+  - override /datamodule/transforms/spectrogram_augmentations: none.yaml
+
+tags: ["BEANS", "embeddings", "multiclass", "k-32"]
+seed: 1
+train: True
+test: True
+
+logger:
+  wandb:
+    tags: ${tags}
+    mode: online
+
+module:
+  optimizer:
+    lr: 1e-2
+    weight_decay: 5e-4
+    
+trainer:
+  limit_val_batches: ${datamodule.val_batches} 
```

Comment:
We probably don't have to limit the trainer anymore if there is no valid split but should be fine anyway!

Reply:
@XgamerTV It is actually still needed to tell the trainer that no validation will happen.